### DISCUSSION TITLE
Projects (beta): deleting issues from a project is hard

### DISCUSSION THREAD

**Discussion by brettgoulder** [community/community#4733] [Projects and Issues]
**Current behavior**
If I focus on one of the "Title" cells that has an issue, I click the delete button on my keyboard, I see a little error message pop up that says "Title cannot be deleted". 

In order to delete an issue from a project, I need to click the down arrow on the left (which is super tiny and can be hard to discover) then click the dropdown to delete the issue from the project.

**Expected behavior**
When I click the delete button on my keyboard when highlighting an issue, I expect the delete modal to pop up and ask me whether I want to delete the issue from the project.

Alternatively, if you want to keep this error/behavior, then atleast make it easier to delete issues from a project. The dropdown button on the left is not easy to discover and the only keyboard shortcut to do this is `cmd-shift-\` which only opens up the row actions menu. There is no keyboard shortcut to remove the issue itself.

**Comment by azenMatt** [comment]
Thanks for the feedback @brettgoulder 

We have a couple of updates coming soon that should help here... 

1. The ability to select an item (row or card) with `shift+space`
2. Hitting the delete key acts as a short cut to the delete command
3. In addition to deleting, you can also archive (shortcut `e`) 
4. You can select multiple items, via either `cmd+click` or `shift+arrow keys` 

The multi-select and delete should be with you next week and then the archive work just behind it!
